web-components/src/components/form/CreditRetireForm.tsx

```diff
@@ -190,6 +191,34 @@ export const CreditRetireFields = ({
   );
 };
 
+const Label = styled('div')(({ theme }) => ({
+  color: theme.palette.primary.contrastText,
+  fontSize: theme.typography.pxToRem(16),
+  marginTop: theme.spacing(8),
+  [theme.breakpoints.up('sm')]: {
+    fontSize: theme.typography.pxToRem(18),
+  },
+}));
+
+const LabelCenter = styled('div')(({ theme }) => ({
+  color: theme.palette.primary.contrastText,
+  fontSize: theme.typography.pxToRem(16),
+  textAlign: 'center',
+  [theme.breakpoints.up('sm')]: {
+    fontSize: theme.typography.pxToRem(18),
+  },
+}));
+
+export const RetirementReminder = ({
+  centered,
+}: {
+  centered?: boolean;
+}): JSX.Element => {
+  const msg = 'Retirement is permanent and non-reversible.';
+  if (centered) return <LabelCenter>{msg}</LabelCenter>;
+  return <Label>{msg}</Label>;
+};
+
```

Comment:
@haveanicedavid What do you think about this? It is the easiest option that has occurred to me, but I was not very clear about it. My initial intention was to create a single Label styled component and pass the variable to display one variant or another? but in the end it seemed simpler like this, but not sure